examples/e-commerce/src/App.vue

```diff
@@ -517,6 +519,10 @@ export default {
         value.max !== null ? value.max : range.max,
       ];
     },
+    isSelectedHitsPerPageValue(itemValue, force = false) {
```

Comment:
Is the `force` argument making the value as default? If so, it seems like this function is addressing several purposes.